python/test/test_publisher.py

```diff
@@ -1315,7 +1315,7 @@ def do_test_publish_check_without_base_stats(self, errors: List[ParseError], ann
                            '3vULqr9z48alUDTb1XTaLTRPlI4YQM0EU2gdwCzIEYwjllAq2P'
                            '1sIUrxjpD1E+YjA0QXwf0TM7hqlgOC5HMP/dt/RevnK18F3THx'
                            'FS08fz1s0zBZBc2w5zHdX73QAAAA==\n'.format(errors='{} errors\u2004\u2003'.format(len(errors)) if len(errors) > 0 else ''),
-                'annotations': annotations
+                'annotations': annotations,
```

Comment:
This `,` is not needed, right? Can it be removed?

Reply:
Yes I think it's left over from unrelated changes, will revert

Reply:
Done